assume-aws-sso-role

```diff
@@ -25,244 +25,242 @@ aws_session_duration=3600
+  echo "$1" | cut -d ':' -f 6 | cut -d '/' -f 2
 }
 
 set_role_arn_from_account_and_role() {
-    aws_role_arn="arn:aws:iam::${aws_account}:role/${aws_role}"
+  aws_role_arn="arn:aws:iam::${aws_account}:role/${aws_role}"
 }
 
 set_account_and_role_from_role_arn() {
-    aws_account=$(account_number_from_role_arn ${aws_role_arn})
-    aws_role=$(role_name_from_role_arn ${aws_role_arn})
+  aws_account=$(account_number_from_role_arn ${aws_role_arn})
+  aws_role=$(role_name_from_role_arn ${aws_role_arn})
 }
 
 role_info_from_profile() {
-    if [ -z "${aws_region}" ]; then
-        aws_region=$(aws configure get region --profile ${aws_profile})
-    fi
-    if [ -z "${aws_region}" ]; then
-        echo_maybe "Region not found in profile ${aws_profile}."
-    fi
-
-    aws_role_arn=$(aws configure get role_arn --profile ${aws_profile})
-    result=$?
-    if [ ${result} -ne 0 ]; then
-        return 1
-    fi
-    set_account_and_role_from_role_arn
+  if [ -z "${aws_region}" ]; then
+    aws_region=$(aws configure get region --profile ${aws_profile})
+  fi
+  if [ -z "${aws_region}" ]; then
+    echo_maybe "Region not found in profile ${aws_profile}."
+  fi
+
+  aws_role_arn=$(aws configure get role_arn --profile ${aws_profile})
+  result=$?
+  if [ ${result} -ne 0 ]; then
+    return 1
+  fi
+  set_account_and_role_from_role_arn
 }
 
 load_credentials_from_file() {
-    credentials=$(cat ${credentials_file})
-    expiration=$(echo ${credentials} | jq -r ".Expiration")
-    expiration_in_seconds=$(date_to_seconds ${expiration})
-    now_in_seconds=$(date "+%s")
-    if [ ${expiration_in_seconds} -gt ${now_in_seconds} ]; then
-        output_from_credentials "${credentials}"
-        echo_maybe "Credentials expires at $(local_time_for ${expiration})"
-        return 0
-    else
-        return 1
-    fi
+  credentials=$(cat ${credentials_file})
+  expiration=$(echo ${credentials} | jq -r ".Expiration")
+  expiration_in_seconds=$(date_to_seconds ${expiration})
+  now_in_seconds=$(date "+%s")
+  if [ ${expiration_in_seconds} -gt ${now_in_seconds} ]; then
+    output_from_credentials "${credentials}"
+    echo_maybe "Credentials expires at $(local_time_for ${expiration})"
+    return 0
+  else
+    return 1
+  fi
 }
 
-
 __usage="Assume a role on an AWS account using the token exchange method. The
 region, account number, and role can be supplied in two different and mutually
 exclusive ways: Either an account number and role is supplied on the command
 line or a profile name referring to a profile in AWS's config file is supplied.
 
 Common options:
 
-    -d: The AWS session duration in seconds. Defaults to 3600 seconds (1 hour).
+-d: The AWS session duration in seconds. Defaults to 3600 seconds (1 hour).
 
-    -e: The region to use. If this is supplied then it overrides any region
-        supplied in the AWS config file (see below).
+-e: The region to use. If this is supplied then it overrides any region
+supplied in the AWS config file (see below).
```

Comment:
I think we should manually restore the indentation of this and similar lines in the help text to keep the text easy to read on the command line.